CMakeLists.txt

```diff
@@ -873,7 +873,7 @@ if(ENABLE_SYCL)
   endif()
   add_library(SYCL::host INTERFACE IMPORTED)
   add_library(SYCL::device INTERFACE IMPORTED)
-  find_package(IntelDPCPP REQUIRED CONFIGS IntelDPCPPConfig-modified.cmake PATHS ${PROJECT_CMAKE})
+  find_package(IntelDPCPP REQUIRED CONFIGS IntelDPCPPConfig-modified.cmake PATHS ${PROJECT_CMAKE} NO_DEFAULT_PATH)
```

Comment:
Not sure why NO_DEFAULT_PATH here.  I think this just prevents finding the package in the location CMake does/will eventually think is standard?  Can't this still be overridden with `-DIntelDPCPP_ROOT=...`

Reply:
Ah I read this too fast, makes sense.